CHANGELOG.md

```diff
@@ -1 +1,99 @@
+
+### Documentation
+* Correct references to --publish long syntax in docs [docker/cli#746](https://github.com/docker/cli/pull/746)
+* Corrected descriptions for MAC_ADMIN and MAC_OVERRIDE [docker/cli#761](https://github.com/docker/cli/pull/761)
+* Updated developer doc to explain external CLI [moby/moby#35681](https://github.com/moby/moby/pull/35681)
+- Fix "on-failure" restart policy being documented as "failure" [docker/cli#754](https://github.com/docker/cli/pull/754)
+- Fix anchors to "Storage driver options" [docker/cli#748](https://github.com/docker/cli/pull/748)
+
+### Logging
+
+* Modify log in order to be same below [moby/moby#35567](https://github.com/moby/moby/pull/35567)
+* Update Graylog2/go-gelf vendoring. Fixes #35613 [moby/moby#35765](https://github.com/moby/moby/pull/35765)
+- Fix awslogs batch size calculation [moby/moby#35726](https://github.com/moby/moby/pull/35726)
+- Fix typo in log-message [moby/moby#35768](https://github.com/moby/moby/pull/35768)
+
+### Networking
+
+* [AUTOMATED] libnetwork Vendoring [moby/moby#35791](https://github.com/moby/moby/pull/35791)
+- Fix external networks in stacks [docker/cli#743](https://github.com/docker/cli/pull/743)
+
+### Runtime
+
+* Carry #35609 (remove import of opencontainers/runc in windows) [moby/moby#35788](https://github.com/moby/moby/pull/35788)
+* Devmapper: Log fstype and mount options during mount error [moby/moby#35732](https://github.com/moby/moby/pull/35732)
+* Windows: Case-insensitive filename matching against builder cache [moby/moby#35793](https://github.com/moby/moby/pull/35793)
+- Fix race conditions in libcontainerd process handling [moby/moby#35809](https://github.com/moby/moby/pull/35809)
+* Use commit-sha instead of tag for containerd [moby/moby#35770](https://github.com/moby/moby/pull/35770)
+* Disallow using legacy (V1) registries [moby/moby#35751](https://github.com/moby/moby/pull/35751)
+* Don't append the container id to custom directory checkpoints. [moby/moby#35694](https://github.com/moby/moby/pull/35694)
+* Re-validate Mounts on container start [moby/moby#35833](https://github.com/moby/moby/pull/35833)
+- Fix overlay2 storage driver inside a user namespace [moby/moby#35794](https://github.com/moby/moby/pull/35794)
+* Zfs: fix busy error on container stop [moby/moby#35674](https://github.com/moby/moby/pull/35674)
+- Fix #35843 regression on health check workingdir [moby/moby#35845](https://github.com/moby/moby/pull/35845)
+- Fix VFS graph driver failure to initialize because of failure to setup fs quota [moby/moby#35827](https://github.com/moby/moby/pull/35827)
+- Fix event filter filtering on "or" [moby/moby#35896](https://github.com/moby/moby/pull/35896)
+
+### Swarm Mode
+
+* Vendor docker/swarmkit to 713d79d [moby/moby#35838](https://github.com/moby/moby/pull/35838)
+* Vendoring swarmkit a6519e28ff2a558f5d32b2dab9fcb0882879b398 [moby/moby#35811](https://github.com/moby/moby/pull/35811)
```

Comment:
Can you list the fixes/changes that this brought in? (see https://github.com/moby/moby/pull/35811 - note that the first one of those was already included in 17.12)